src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -99,7 +100,8 @@ public FindInFilesDialog(OperationWithInput<State> operation)
       dirChooser_ = new DirectoryChooserTextBox("Search in:", null);
       dirChooser_.setText("");
       mainWidget_ = GWT.<Binder>create(Binder.class).createAndBindUi(this);
-
+      labelSearchPattern_.setFor(txtSearchPattern_);
+      labelFilePatterns_.setFor(listPresetFilePatterns_);
```

Comment:
This is a recurring pattern -- is there some way to make this declarative in the `UiBinder`? If memory serves container widgets are a thing you can create, so we could make a widget that grouped together a form control and its label such that they were wired together for accessibility declaratively.

Reply:
You can also just nest an element inside a label and the association is automatic (don't have to use the for attribute in that case). Juggling doing that vs. trying not to mess with existing layouts. Either way, yes, this is something I'm going to try and make cleaner.